### DISCUSSION TITLE
getCameraTarget() getting other units besides meters

### DISCUSSION THREAD

**Discussion by Paolo-Sotorro** [google/model-viewer#3317] [Q&A]
Hi,

Short question - getCameraTarget() is returning values in meters, is there an option to get values in other units (cm, mm) as well?

**Comment by elalish** [comment]
No, but you can multiply by 100 or 1000 :)

**Reply by Paolo-Sotorro** [reply]
How is this helpful?! 
I need to get values from getCameraTarget() like 35cm and it will return 0 or 1m. 
Setting the values is not a problem but reading is.

**Reply by elalish** [reply]
What? It should return 0.35m. Do you have an example where the answer is being rounded?
